config.yml.example

```diff
@@ -6,7 +6,7 @@ user: git
 # You only have to change the default if you have configured Unicorn
```

Comment:
That should work. We'd need to make that in the Dockerfile though, as the existing hosted SaaS platform uses this repo too, and having a `config.yml` file existing here might cause problems for that. I guess this could be done easily in the Dockerfile: `cp config.yml.example config.yml`.